internal/kafka/command_link_describe.go

```diff
@@ -70,5 +146,39 @@ func newDescribeLink(link kafkarestv3.ListLinksResponseData, topic string) *desc
 		State:                link.GetLinkState(),
 		Error:                linkError,
 		ErrorMessage:         link.GetLinkErrorMessage(),
+		Tasks:                tasks,
+	}
+}
+
+func toTaskOut(tasks []kafkarestv3.LinkTask) []serializedTaskOut {
+	var tasksToEncode []kafkarestv3.LinkTask
+	if tasks != nil {
+		tasksToEncode = tasks
+	} else {
+		tasksToEncode = make([]kafkarestv3.LinkTask, 0)
+	}
```

Comment:
I don't think we necessarily need a separate variable here.
```suggestion
	if tasks == nil {
		tasks = make([]kafkarestv3.LinkTask, 0)
	}
```

Reply:
Nice, even better!